src/github/deployment.ts

```diff
@@ -70,8 +70,23 @@ export const processDeployment = async (
 
 	const result: DeploymentsResult = await jiraClient.deployment.submit(jiraPayload);
 	if (result.rejectedDeployments?.length) {
+
+		let extraDebugInfo = {};
+		try {
+			const deploymentData = jiraPayload.deployments.length > 0 ? jiraPayload.deployments[0] : undefined;
```

Comment:
Is this temporary while debugging or permanent? If it's the latter, I say we pull this logic into a function, export it, and add a unit test.

Reply:
Good point, let me pull that into a separate function. Tbh it is for debugging only but I suspect it could be left there for long time.